package.json

```diff
@@ -5,9 +5,9 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start",
+    "start": "npx serve ./out/",
```

Comment:
Why wont `next start` work over here?

Reply:
In my opinion let's just have next start cause the website is not entirely static / server side rendered

Reply:
After did the changes I will rollback to use export again after it breaks the staging build, looks like Netlify has their own command as referred to the image below
![Screenshot 2024-06-12 172329](https://github.com/asyncapi/conference-website/assets/54782057/a26f9ec2-824a-4b03-a854-6511bfd066ff)

Reply:
I see let's get @AceTheCreator opinion on this. Cause I don't have access to netlify

Reply:
Hey @jerensl 

have a look at the Netlify.toml, the Node version is not as said and you can change the build and deploy command from there I guess.

So look into it :)